other_tutorials/wca-dimer_openmm/we_custom/env.sh

```diff
@@ -1,5 +1,5 @@
 # Inform WEST where to find Python and our other scripts where to find WEST
```

Comment:
We don't need the `WEST_ROOT` or `WEST_PYTHON` environmental variables with westpa-2.0. I think we should remove them. We would then need to go into files like `run.sh` and change `$WEST_ROOT/bin/w_run` to just `w_run`. This makes the assumption that you are in the proper conda env or virtual environment. I'm not 100% sure how we should handle that because the activation of the env will be user specific. 

@atbogetti -- do you have any ideas of how we should formalize this?

Reply:
@synapticarbors I think that the `env.sh` could be reserved for executable path definitions (such as `SANDER=/usr/bin/sander`) that we recommend for dynamics programs.  Could `$WEST_SIM_ROOT` also be hard-coded in to just be `pwd` because I think most of the time that is the case by default.  Maybe env.sh wouldn't even be needed then.